jquery/index.js

```diff
@@ -1,2 +1,15 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
```

Comment:
I would think we want to set this to 2016, since that's the first year this file was worked on?  Similar comment for all new headers in this PR

Reply:
Yeah, I called out this issue [here](https://github.com/okta/legal-linter/pull/1#discussion_r181281343) - but thought the conversations with Legal resulted in not having a preference. Moving forward, all files w/o a header should have have the **start date** be the day the year the file was created?

Reply:
Using the first edit date of the file makes the most sense (to me) from the perspective of how copyright works.  If the linter were to help with this it would need to look at files on a per-file basis (global configuration would be assuming all files were created in the same year, which may not be true)